vim-patch:e84d2d4432cd

clason opened the pull request:
runtime(sh): Update ftplugin, fix vim/vim#14101 (vim/vim#14102)

Add the 'b' flag to 'comments', so that the shebang line is not detected as comment.

https://github.com/vim/vim/commit/e84d2d4432cd6e43f2bb300d02abc90d551bcf4a

Co-authored-by: dkearns <dougkearns@gmail.com>

seandewar commented:
Commit is empty :scream:

clason commented:
vim runtime is perfect 😤

seandewar reviewed (approved):
The commit message seems to be referencing Neovim issues for some reason. :eyes:

clason commented:
Yeah; the commit messages don't translate well across repos. The vim-patch script needs some adaptation to the new world we live in.

zeertzjq commented:
Oh, I broke the script again

clason committed:
vim-patch:e84d2d4432cd

runtime(sh): Update ftplugin, fix vim/vim#14101 (vim/vim#14102)

Add the 'b' flag to 'comments', so that the shebang line is not detected as comment.

Fixes vim/vim#14101.

https://github.com/vim/vim/commit/e84d2d4432cd6e43f2bb300d02abc90d551bcf4a

Co-authored-by: dkearns <dougkearns@gmail.com>

clason commented:
Seems to work!
